examples/closures/closures.bal

```diff
@@ -35,7 +35,7 @@ function multilevelClosure() returns (function (int) returns int) {
 }
 
 // Example showing how function pointers are passed around with closures
-// and inner scope lambdas access the outer scope variables.
+// and inner scope anonymous function access the outer scope variables.
```

Comment:
Shall we change as follows:
"// Example to represents how function pointers are passed with closures
 // so that the inner scope anonymous function can access the outer scope variables."